File: SDKs/github.com/ghes-2.19/1.1.4/go/pkg/models/shared/blob.go
package shared

type Blob struct {
	Content            string  `json:"content"`
	Encoding           string  `json:"encoding"`
	HighlightedContent *string `json:"highlighted_content,omitempty"`
	NodeID             string  `json:"node_id"`
	Sha                string  `json:"sha"`
	Size               int64   `json:"size"`
	URL                string  `json:"url"`
}

File: SDKs/github.com/ghes-2.20/1.1.4/typescript/src/sdk/models/shared/blob.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// Blob
/** 
 * Blob
**/
export class Blob extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=content" })
  content: string;

  @SpeakeasyMetadata({ data: "json, name=encoding" })
  encoding: string;

  @SpeakeasyMetadata({ data: "json, name=highlighted_content" })
  highlightedContent?: string;

  @SpeakeasyMetadata({ data: "json, name=node_id" })
  nodeId: string;

  @SpeakeasyMetadata({ data: "json, name=sha" })
  sha: string;

  @SpeakeasyMetadata({ data: "json, name=size" })
  size: number;

  @SpeakeasyMetadata({ data: "json, name=url" })
  url: string;
}
